### ISSUE TITLE
Re-opening an outline with an @auto-md tree loses clones

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3355]
Re-opening an outline that contains an `@auto-md` tree changes clone nodes to non-clones, but only for clones of the `@auto-md` tree.  Other clones remain clones.  Not tested with other types of `@auto-xx` trees.

See Google Groups thread [
@auto-ms does not get along with Leo's clones
](https://groups.google.com/g/leo-editor/c/B9ibKxY8MRE)

**Comment by largo84** [comment]
As I understand it, @auto-xx files w/ clones *should be* maintained when there's an @persistence tree. There's a setting to create one that's off by default. However, none of my @auto-xx files retain clones even w/ the @persistence node in place. This has been the case for quite some time. I seem to recall that it worked before, but don't remember when it stopped working.

**Comment by tbpassin** [comment]
Ah, the `@persistence` nodes have always been a mystery to me, and I don't think I have them enabled.  I should check.

**Comment by LewisNeal** [comment]
Previous discussion - Losing clones after restart
https://groups.google.com/g/leo-editor/c/oKGhRhUVTXA/m/xqUHMJzWAQAJ

**Comment by tbpassin** [comment]
I just turned on `@bool create-at-persistence-nodes-automatically = True` and now the clones don't disappear.  I've only tried one round of save/reload. but so far so good.

**Comment by tbpassin** [comment]
After a few more tests, with `@bool create-at-persistence-nodes-automatically = True` clones seem to be saved properly in my `@auto-md` test case.  Of course I haven't had a chance to work on a real, complex external file, and I haven't tried turning the setting off and manually adding an `@persistence` node.  I'll try that next.

**Comment by tbpassin** [comment]
All right, I'm satisfied.  I turned off the `@bool create-at-persistence-nodes-automatically` setting and deleted the `@persistence` node.  Then I manually inserted a new `@persistence` node, created some clones of nodes in the `@auto-md` tree, saved and re-opened the outline.  The clones were remembered correctly and the `@persistence` node was now a tree and seems to have been populated correctly.

So I'm going to close this issue.  It's still possible that the original poster in the Groups thread has a different problem from this.
